twilio/rest/accounts/v1/credential/aws.py

```diff
@@ -86,13 +86,9 @@ def page(self, page_token=values.unset, page_number=values.unset,
         :returns: Page of AwsInstance
         :rtype: twilio.rest.accounts.v1.credential.aws.AwsPage
         """
-        params = values.of({'PageToken': page_token, 'Page': page_number, 'PageSize': page_size, })
+        data = values.of({'PageToken': page_token, 'Page': page_number, 'PageSize': page_size, })
 
-        response = self._version.page(
-            'GET',
-            self._uri,
-            params=params,
-        )
+        response = self._version.page(method='GET', uri=self._uri, params=data, )
```

Comment:
Why the name change?

Reply:
Mainly a side effect of refactoring. It's just an internal variable name, so it shouldn't matter.